05_stringer/marksomething/main.go

```diff
@@ -0,0 +1,20 @@
+package main
+
+import (
+	"fmt"
+	"io"
+	"os"
+)
+
+var out io.Writer = os.Stdout
+
+// IPAddr is an IPv4 Address
+type IPAddr [4]byte
+
+func (a IPAddr) String() string {
```

Comment:
This is only very minor, I like to use argument name that is a bit more obvious, in this case, it's fine using 'a', but a larger function with more arguments might make be less obvious. (this is a comment, not necessarily a request for a change though).

Reply:
@willshen8, much of the Go literature encourages a style where receiver names are 1 or 2 characters long and it is a convention that is carried across most of the Go code you'll read.

Andrew Gerrand (our day 1 guest speaker) has presented on this before: https://talks.golang.org/2014/names.slide#11